README.md

```diff
@@ -25,6 +25,7 @@ $ composer require php-http/httplug
 
 This is the contract package for HTTP Client. Use it to create HTTP Clients which are interoperable and compatible with [PSR-7](http://www.php-fig.org/psr/psr-7/).
 
+This library is the official successor of the [ivory http adapter](https://github.com/egeloen/ivory-http-adapter) and the code is based on the ivory http adapter code.
```

Comment:
Ivory http adapter and code expressions are repeated here. Can we make it simpler? Also, the code is completely refactored. Actually it has been refactored several times. I think it is enough to mention that the idea is from ivory, it evolved from an adapter package to a PSR-7 HTTP Client abstraction and it is a successor of IHA.